docs_devel/HowToCreateFilterPlugin.md

```diff
@@ -1,314 +0,0 @@
-# How to create a file filter plugin for OmegaT
```

Comment:
Instead of deleting this file completely, could you replace the contents with a pointer to the plugin project on GitHub? Something simple like

```
This document has moved to:
https://...
```

Reply:
It is over action to remove How-to document here. I'd like to restore it.